### PR TITLE
add fusionfeed data page

### PR DESCRIPTION
**Pull request by ccbrown**
Adds a "data" page, which takes things a step farther than the bullet points on the FusionFeed index and answers a lot of common questions.

To review the pretty version of the page: https://github.com/tempus-ex/docs/blob/ff-data-page/content/fusionfeed/data.mdx

### CODE REVIEW

File: content/fusionfeed/data.mdx [hunk 1]
```diff
@@ -0,0 +1,104 @@
+---
+title: Data
+---
+# Data
+
+FusionFeed provides comprehensive coverage for foundational game data. Unless otherwise mentioned,
+all data is made available in realtime at low latency during the game.
+
+FusionFeed data is ingested from the truest and lowest latency source possible, usually on-site.
```
Comment (ashukur) [inline content/fusionfeed/data.mdx:9]: Does this statement hold up for things like GSIS, or if we have to integrate with Opta in the future?

File: content/fusionfeed/data.mdx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+- **8K Cameras** - We set up fixed 8K cameras around the stadium to capture all the action that's not
+  caught by TV cameras.
+- **FusionCams** - These are virtual cameras that follow a specific subject, region, or group of
+  subjects within the field of view of an 8K camera. Users can create FusionCams for use-cases such
+  as isolating individual players or as an alternative to the traditional all 22 cameras.
+
+Video is made available through FusionFeed via the HLS protocol. HLS video can be distributed at a
+global scale directly from FusionFeed with approximately 10 seconds of end-to-end latency.
+
+On request, SRT feeds can be provided for video for viewing with sub-second latency from the camera
```
Comment (ashukur) [inline content/fusionfeed/data.mdx:46]: might need more clarification that this is a special request on a case by case basis

**Review by ashukur — APPROVED** [review]
couple comments but this will be a big help for customers

File: content/fusionfeed/data.mdx (hunk 1 above)
Comment (ccbrown) [reply content/fusionfeed/data.mdx:9]: The first part does. GSIS is the truest and lowest latency source of official stats. It's cloud-based, so the second part isn't true of GSIS, but I think it's fair to say we usually ingest on-site if it's possible.

**Pull request merged**